Header header

uint32 x_size
uint32 y_size
geometry_msgs/Point[] points
